index.js

```diff
@@ -72,9 +80,26 @@ const argv = require('yargs')
     .help()
     .argv;
 
+async function _setViewPort(page, argv) {
+    const w = argv.viewportWidth;
+    const h = argv.viewportHeight;
```

Comment:
I prefer to be consistent with naming values, avoid abbreviations, and avoid having copies of the same value in different places

How about we do something more like: 

```js
async function _setViewPort(page, { viewportWidth, viewportHeight }) {
```